docs/content/guides/developer/getting-started/graphql-rpc.mdx

```diff
@@ -368,17 +772,203 @@ mutation ($tx: String!, $sigs: [String!]!) {
+
 :::
+-->
+
+## Migrating to GraphQL from JSON-RPC {#migrate}
+
+The examples in the following sections demonstrate the typical API calls. The calls are tabbed to show both the JSON-RPC call and the equivalent GraphQL structure. For a comprehensive list of all available GraphQL features, consult the [reference](../../../references/sui-graphql).
+
+### Example 1: Get total transaction blocks
+
+Get the total number of transaction blocks in the network.
+
+<Tabs groupId="migrating-json-rpc">
+
+<TabItem value="json-rpc1" label="JSON-RPC">
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "sui_getTotalTransactionBlocks",
+  "params": []
+}
+```
+
+</TabItem>
+<TabItem value="graphql1" label="GraphQL">
+
+```graphql
+query {
+  checkpoint {
+    networkTotalTransactions
+  }
+}
+```
+
+</TabItem>
+</Tabs>
+
+### Example 2: Get a specific transaction block
+
+Get the transaction block by its digest.
+
+<Tabs groupId="migrating-json-rpc">
+
+<TabItem value="json-rpc2" label="JSON-RPC">
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "sui_getTransactionBlock",
+  "params": [
+    "Hay2tj3GcDYcE3AMHrej5WDsHGPVAYsegcubixLUvXUF",
+    {
+      "showInput": true,
+      "showRawInput": false,
+      "showEffects": true,
+      "showEvents": true,
+      "showObjectChanges": false,
+      "showBalanceChanges": false
+    }
+  ]
+}
+```
+
+</TabItem>
+<TabItem value="graphql2" label="GraphQL">
+
+```graphql
+query {
+  transaction(digest: "Hay2tj3GcDYcE3AMHrej5WDsHGPVAYsegcubixLUvXUF") {
+    gasInput {
+      gasSponsor {
+        address
+      }
+      gasPrice
+      gasBudget
+    }
+    effects {
+      status
+      timestamp
+      checkpoint {
+        sequenceNumber
+      }
+      epoch {
+        epochId
+        referenceGasPrice
+      }
+    }
+  }
+}
+```
+</TabItem>
+</Tabs>
+
+### Example 3: Get coin objects owned by an address
+
+Return all `Coin<0x2::sui::SUI>` objects that an address owns.
+
+<Tabs groupId="migrating-json-rpc">
+
+<TabItem value="json-rpc3" label="JSON-RPC">
+
+```json
+query {
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "suix_getCoins",
+  "params": [
+    "0x5094652429957619e6efa79a404a6714d1126e63f551f4b6c7fb76440f8118c9", //owner
+    "0x2::sui::SUI",                                                      //coin type
+    "0xe5c651321915b06c81838c2e370109b554a448a78d3a56220f798398dde66eab", //cursor
+    3 //limit
+  ]
+}
+```
+
+</TabItem>
+<TabItem value="graphql3" label="GraphQL">
+
+```graphql
+query {
+  address(address: "0x5094652429957619e6efa79a404a6714d1126e63f551f4b6c7fb76440f8118c9") {
+    coins(
+      first: 3,
+      after: "IAB3ha2PEA4ESRF4UErsJufJEwYpmSbCq7UNpxIHnLhG",
+      type: "0x2::sui::SUI"
+    ) {
+      nodes {
+        address
+      }
+    }
+  }
+}
+```
+
+:::info
+
+The cursor is now passed in the `after` (or `before`) fields on the connection, and the limit in the `first` or `last` fields.
+
+:::
+
+</TabItem>
+</Tabs>
+
+## New features
```

Comment:
Couple more new features we can highlight:

- Time travel queries with `checkpoint { query { ... } }`
- Nested dynamic field look-ups (and paginating dynamic fields with their contents in one request).
- Fetching live objects by owner kind and type (e.g. all shared objects of a certain type).
- Fetching deserialized input and output object contents from a transaction that has just executed.